pex/jobs.py

```diff
@@ -732,7 +732,7 @@ def iter_map_parallel(
     # least two slots to ensure we process input items in parallel.
     pool_size = max(2, min(len(input_items) // min_average_load, _sanitize_max_jobs(max_jobs)))
 
-    perform_install = functools.partial(_apply_function, function)
+    apply_function = functools.partial(_apply_function, function)
```

Comment:
This was an, in the end, unrelated re-name. I was poking around in the resolve / wheel install machinery looking at timings and narrowing in on the double-hashing going on.